src/DependencyInjection/JsonRpcMethodDefinitionHelper.php

```diff
@@ -26,8 +43,26 @@ public function findAndValidateJsonRpcMethodDefinition(ContainerBuilder $contain
             $this->validateJsonRpcMethodDefinition($serviceId, $container->getDefinition($serviceId));
 
             foreach ($tagAttributeList as $tagAttributeKey => $tagAttributeData) {
-                $this->validateJsonRpcMethodTagAttributes($serviceId, $tagAttributeData);
-                $methodName = $tagAttributeData[JsonRpcHttpServerExtension::JSONRPC_METHOD_TAG_METHOD_NAME_KEY];
+                try {
+                    $this->validateJsonRpcMethodTagAttributes($serviceId, $tagAttributeData);
+                    $methodName = $tagAttributeData[JsonRpcHttpServerExtension::JSONRPC_METHOD_TAG_METHOD_NAME_KEY];
+                } catch (\LogicException $e) {
+                    // Annotation routing loader is conditionally added by the framework
+                    // bundle; if it's enabled, allow for our own annotation-based
+                    // method discovery.
+                    if ($this->annotationsEnabled) {
+                        $reflection = $container->getReflectionClass($container->getDefinition($serviceId)->getClass());
+                        $annotation = $this->reader->getClassAnnotation($reflection, self::METHOD_ANNOTATION_CLASS);
+                        if ($annotation) {
```

Comment:
```suggestion
                        $annotation = $this->reader->getClassAnnotation($reflection, Annotation\JsonRpcMethod::class);
                        if (null !== $annotation) {
```
And add `use Yoanm\SymfonyJsonRpcHttpServer\Annotation;` on top of file (I prefer to keep `Annotation\` just to avoid misunderstood between JsonRpc method class and the annotation class).

You can remove the constant then, I don't think it brings any advantage.